### ISSUE TITLE
.github/workflow files are ignored by default

### ISSUE THREAD

**Issue by Snailedlt** [Logerfo/newline-action#66]
At the moment my workflow file is being ignored:
![image](https://user-images.githubusercontent.com/43886029/211861954-a55a27bb-5c0f-4b6c-8ca2-14c4f3733f94.png)

This shouldn't happen because GitHub still says `No newline at end of file` for that file, so it makes no sense to ignore it by default.
![image](https://user-images.githubusercontent.com/43886029/211861613-8c879203-2594-452e-be5f-fd2ed6cc779c.png)

**Comment by Logerfo** [comment]
Interesting... I'd say this is a bug. There's no specified behavior for skipping other than the assigned paths to ignore. If no assignment is made, the default configuration ignores `bin/**`, `node_modules/**` and `out/**`, none of which should ignore any workflow file. I don't know if the actual behavior is related to the fact that the skipped file is the running workflow file itself.

**Comment by Snailedlt** [comment]
Hmm, I first thought it might be a permission problem, but then the `is ignored` message seems weird, cause that only happens if the if statement above it is true.

Definitely a bug I'd say. I'm soon updating another workflow file, so I'll let you know how that goes :)

**Comment by Logerfo** [comment]
@Snailedlt is it possible that your token has no read permission for the workflow file, but has for some files? I don't know if that depth of granularity is supported, but perhaps you'd know if you use it.

**Comment by Snailedlt** [comment]
I don't think that could even be the cause of the issue, could it?

As far as I can tell that wouldn't affect this if statement, which is what is ran to return that message you see in the first image in the issue description.

https://github.com/Logerfo/newline-action/blob/d5cb021020d383c7f38802dce86ce4bf6f14a74f/src/main.js#L114-L117

**Comment by Logerfo** [comment]
@Snailedlt if the blob object is unaware of the existence of the workflow file, I think that might cause the problem you stumbled upon.

**Comment by Snailedlt** [comment]
I confirmed that it's at least inside .github and .github/workflow folders. Could be permission like you say, any idea how to fix it?
![image](https://user-images.githubusercontent.com/43886029/211906576-c70c3e01-6659-446a-a3d6-686185627f32.png)

**Comment by Logerfo** [comment]
@Snailedlt can you confirm that the action is successfully reading files located in directories other than root?

**Comment by Snailedlt** [comment]
Not at the moment because we have to focus on other issues. But once I get around to doing it, I'll let you know :)

**Comment by Logerfo** [comment]
@Snailedlt I was wondering if the implemented blob was wrong, maybe only reading the root folder, but it's working fine for me: https://github.com/Logerfo/newline-test/actions/runs/3932390852/jobs/6724891213. Maybe it really is something related to permissions, specially if your repo is in an organization, but I don't know much about GitHub's granularity.

**Comment by Snailedlt** [comment]
Yes, that's beginning to look more and more likely. The project is in a private repo in an organization

**Comment by pablogamboa** [comment]
I'm getting the same issue, the project is a private repo in an organization and it skips the files
